src/main/java/com/mongodb/kafka/connect/sink/converter/LazyBsonDocument.java

```diff
@@ -185,4 +193,15 @@ private Object writeReplace() {
   private void readObject(final ObjectInputStream stream) throws InvalidObjectException {
     throw new InvalidObjectException("Proxy required");
   }
+
+  private static String unambiguousToString(@Nullable final Object v) {
+    String vToString = String.valueOf(v);
+    if (v == null) {
+      return format("'%s' (null reference)", vToString);
+    } else if (vToString.equals(String.valueOf((Object) null))) {
+      return format("'%s' (%s, not a null reference)", vToString, v.getClass());
+    } else {
+      return format("'%s' (%s)", vToString, v.getClass());
```

Comment:
Recommend: `v.getClass().getSimpleName()` or `v.getClass().getName()`

Reply:
1. `getSimpleName` erases the package data and returns an empty string for anonymous classes. Given this, `toString`/`getName` is much better for this use case.
2. The only difference between `getName` and `toString` is that `toString` additionally specifies whether it's an interface or a class. Why do you think not having that information is preferable?

Reply:
It's only a recommendation. 

Only the following types are supported:

 - Schema'd data - `Struct`
 - Json data - `Map<String, Object>`
 - Raw json string - `String`
 - Raw bytes = `byte[]`

So there are no cases where anonymous classes could be used and or where package data is particularly helpful. Simple and clear error messages help users, too verbose or complex ones can hinder users.

Reply:
Used `Class.getName`.